mycore-pi/src/main/java/org/mycore/pi/doi/MCRDOIRegistrationService.java

```diff
@@ -71,10 +70,12 @@
 
     private static final int MAX_URL_LENGTH = 255;
 
-    private static final String DATACITE_SCHEMA_PATH = "xsd/datacite/metadata.xsd";
+    private static final String DEFAULT_DATACITE_SCHEMA_PATH = "xsd/datacite/v3/metadata.xsd";
```

Comment:
meanwhile we have version 4. Perhaps we should not put this specification in java code? (could be changed in mycore master ...)